Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -384,6 +436,14 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
             break;
 
         case WI.InlineSwatch.Type.Variable: {
+            if (this._subType) {
+                let palette = this._colorPalette.find(palette => value.toString().slice(4, value.length - 1) == palette.colorSource || value.toString() === palette.colorSource);
```

Comment:
style: we always wrap the parameter of an arrow function in `(` `)`
style: we always use `===`